src/clusterfuzz/_internal/tests/appengine/handlers/upload_testcase_test.py

```diff
@@ -586,3 +586,34 @@ def test_trusted_uploader_post(self):
         'uploader_email': 'uploader@email',
         'window_argument': ''
     }, testcase._to_dict())
+
+    metadata = data_types.TestcaseUploadMetadata.query(
+        data_types.TestcaseUploadMetadata.testcase_id ==
+        testcase.key.id()).get()
+    self.assertIsNotNone(metadata)
+    self.assertDictEqual({
+        'additional_metadata_string': None,
+        'blobstore_key': 'blob_key',
+        'bot_name': None,
+        'bug_information': '',
+        'bug_summary_update_flag': False,
+        'bundled': False,
+        'duplicate_of': None,
+        'filename': 'input',
+        'original_blobstore_key': 'blob_key',
+        'path_in_archive': None,
+        'quiet_flag': False,
+        'retries': None,
+        'security_flag': False,
+        'status': 'Pending',
+        'testcase_id': 5,
+        'timeout': 0,
+        'timestamp': datetime.datetime(2021, 1, 1, 0, 0),
+        'uploader_email': 'uploader@email'
+    }, metadata._to_dict())
+    self.assertEqual({
```

Comment:
Would using `assertDictEqual` (like the one above) here [make debugging simpler](https://stackoverflow.com/questions/34414326/why-is-assertdictequal-needed-if-dicts-can-be-compared-by)?